CONTRIBUTING.md

```diff
@@ -21,3 +23,11 @@ To run the unit tests, just execute:
 ```
 npm test
 ```
+
+## Linting the SCSS files
+
+To lint an SCSS file, run:
+
+```
+scss-lint src/<filename>.scss
```

Comment:
The task for SCSS linting already exist and is easy to run:

```
npm run scsslint
```

Let's change to it. There's already a plan to shift from RubyGems to Node.js based SCSS linter anyway, see #49.